parcel_test.go

```diff
@@ -30,58 +32,139 @@ func getTestParcel() Parcel {
 
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
-	// prepare
-	db, err := // настройте подключение к БД
+	db, errDB := sql.Open("sqlite", "tracker.db")
+
+	require.Nil(t, errDB)
```

Comment:
Есть специальная функция для ошибок
```suggestion
	require.NoError(t, err)
```

Reply:
Исправь, пожалуйста, везде